lib/web/apiserver.go

```diff
@@ -300,6 +300,10 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*APIHandler, error) {
 	// Unauthenticated access to the message of the day
 	h.GET("/webapi/motd", httplib.MakeHandler(h.motd))
 
+	// Unauthenticated access to retrieving the script used to install
+	// Teleport
+	h.GET("/webapi/install-teleport.sh", httplib.MakeHandler(h.installer))
```

Comment:
Can we tuck this under "scripts" or something, and also do we need `.sh` suffix? `/webapi/scripts/installer`.